includes/classes/ExternalConnections/WordPressExternalConnection.php

```diff
@@ -265,6 +265,9 @@ public function remote_get( $args = array() ) {
 			$posts_url = untrailingslashit( $types_urls[ $post_type ] ) . '/?' . $args_str;
 		}
 
+		// Add request parameter to specify Distributor request
+		$posts_url = add_query_arg( 'distributor_request', '1', $posts_url );
```

Comment:
@Ritesh-patel won't this cause distributor to use raw content for all external connection pulls? i'm not sure we want to do that - for sites that aren't using gutenberg, we typically want the _rendered_ content because we can't be sure both sites use the same theme/plugins eg support the same embeds/shortcodes.

Reply:
See https://github.com/10up/distributor/blob/d7a1d90bc8ac15c9296f237524939b558ef8b030/includes/classes/InternalConnections/NetworkSiteConnection.php#L88 for how we do this on push

Reply:
@adamsilverstein thanks for checking this, I was under assumption that this needs to happen for both Classic and Gutenberg editor.